dashboard/app/controllers/pd/user_admin_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Pd::UserAdminController < ApplicationController
+  authorize_resource class: :pd_user_admin
```

Comment:
Actually, `authorize_resource` is preferred here. `authorize!` is the underlying method that doest the authorization from within an action. `authorize_resource` is a class method that simplifies by defining a before_action callback which in turn calls `authorize!`. See https://github.com/CanCanCommunity/cancancan